### PR TITLE
Update BigQuery details on Domain Restriction

### PR DESCRIPTION
**Pull request by kurbycchua**
1. Added Mixpanel BQ Domain and Customer ID see details:

- https://mixpanel.slack.com/archives/C05D3M6547Q/p1690212004457249?thread_ts=1689927975.370529&cid=C05D3M6547Q

- https://mixpanel.slack.com/archives/C04U70H63/p1704697422012679

2. Minor formatting changes

### CODE REVIEW

File: pages/docs/data-pipelines/integrations/bigquery.md [hunk 1]
```diff
@@ -81,27 +81,34 @@ Counting number of times a user has done a specific behavior is also possible by
...
 
-We need two permissions to manage the dataset. 
+   ![image](/230698727-1216833e-8321-46de-a388-8b554a00938c.png)
 
-**BigQuery Job User**
-  * Go to **IAM &Admin** in your Google Cloud Console.
-  * Click **+ ADD** to add principals
-  * Add new principle "export-upload@mixpanel-prod-1.iam.gserviceaccount.com" and set role as "BigQuery Job User", and save.
-![image](/230698732-4dadbccf-1eeb-4e64-a6c7-8926eb49e5cc.png)
+<br />
```
Comment (mavlee) [inline pages/docs/data-pipelines/integrations/bigquery.md]: i think even if the spacing is kinda nice, we're better off getting rid of these random tags, we should be addressing this through css where we can

and actually it looks relatively ok even without it

File: pages/docs/data-pipelines/integrations/bigquery.md [hunk 2]
```diff
@@ -81,27 +81,34 @@ Counting number of times a user has done a specific behavior is also possible by
...
+    
+   **BigQuery Data Owner**
+     * Go to **BigQuery** in your Google cloud Console.
+     * Open the dataset you want mixpanel to export to.
+     * Click on **sharing** and **permissions** in the drop down. 
+     * In the Data Permissions window click on **Add Principal** 
+     * Add new principle "export-upload@mixpanel-prod-1.iam.gserviceaccount.com" and set role as "BigQuery Data Owner", and save.    
+       ![image](/230698735-972aedb5-1352-4ebc-82c4-ef075679779b.png)
+
+<br />
```
Comment (mavlee) [inline pages/docs/data-pipelines/integrations/bigquery.md]: same as above

File: pages/docs/data-pipelines/integrations/bigquery.md (hunk 1 above)
Comment (kurbycchua) [reply pages/docs/data-pipelines/integrations/bigquery.md]: updated

File: pages/docs/data-pipelines/integrations/bigquery.md (hunk 2 above)
Comment (kurbycchua) [reply pages/docs/data-pipelines/integrations/bigquery.md]: updated

**Pull request merged**